### ISSUE TITLE
GTB card has silence immunity 

### ISSUE THREAD

**Issue by anyamichael** [HerculesWS/Hercules#2605]
Golden thief bug card script has wrong script 
	//Status that are blocked by Golden Thief Bug card or Wand of Hermod
	if (status->isimmune(bl))
		switch (type) {
		case SC_DEC_AGI:
		case **SC_SILENCE:**
		case SC_COMA:
		case SC_INC_AGI:
		case SC_BLESSING:
		case SC_SLOWPOISON:
		case SC_IMPOSITIO:
		case SC_LEXAETERNA:
		case SC_SUFFRAGIUM:
		case SC_BENEDICTIO:
		case SC_PROVIDENCE:
		case SC_KYRIE:
		case SC_ASSUMPTIO:
		case SC_ANGELUS:
		case SC_MAGNIFICAT:
		case SC_GLORIA:
		case SC_WINDWALK:
		case SC_MAGICROD:
		case SC_ILLUSION:
		case SC_STONE:
		case SC_QUAGMIRE:
		case SC_NJ_SUITON:
		case SC_SWING:
			return 0;
		}
it has silence
![Screen Shot 2020-01-06 at 18 03 29](https://user-images.githubusercontent.com/59560882/71812437-27e74000-30b2-11ea-9e99-15607ab9b8ea.png)

**Comment by hemagx** [comment]
Confirmed, this also affects many other skills.

**Comment by hemagx** [comment]
I will compile full list soon for pre-re/re

**Comment by Xelliekins** [comment]
Stone Curse and Coma are also status effects that are independent of skills and shouldn't be blocked by card/weapon sources or tarot.

**Comment by GreenStage** [comment]
any news on this?

**Comment by Playtester** [comment]
Might want to mark this as duplicate of: https://github.com/HerculesWS/Hercules/issues/2913

**Comment by skyleo** [comment]
This should be fixed since https://github.com/HerculesWS/Hercules/pull/3155
